### ISSUE TITLE
Wolfenstein (10170)

### ISSUE THREAD

**Issue by davegorst** [ValveSoftware/Proton#5338]
# Compatibility Report
- Wolfenstein (2009)
- 10170

## System Information
- AMD Radeon RX 6800 XT (sienna_cichlid, LLVM 13.0.0, DRM 3.41, 5.13.0-21-generic)
- Proton experimental

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

[steam-10170.log](https://github.com/ValveSoftware/Proton/files/7581734/steam-10170.log)

[gist.txt](https://github.com/ValveSoftware/Proton/files/7581751/gist.txt)

## Symptoms 
CEG DRM appears to be failing. On one or two attempts since introducing CEG support it appears to have loaded fine however it no longer seems to - a command prompt briefly flashes up before it fails (this looks like it's in the log file above).

## Reproduction

Start as multiplayer. Loads and runs fine.
Start as singleplayer - instantly crashes on loading.

**Comment by kisak-valve** [comment]
Hello @davegorst, at first glance, this game doesn't appear to be using CEG.

**Comment by davegorst** [comment]
You're right - it did briefly work after the CEG update, along with a few others, so I might have jumped the gun there. A bit of research and it could be SecuROM (https://support.securom.com/pop_wolfenstein.html)?
